src/py/flwr/server/grpc_server/grpc_server.py

```diff
@@ -73,7 +111,33 @@ def start_insecure_grpc_server(
     servicer = fss.FlowerServiceServicer(client_manager)
     transport_pb2_grpc.add_FlowerServiceServicer_to_server(servicer, server)
 
-    server.add_insecure_port(server_address)
+    if ssl_files is not None:
+        if not valid_ssl_files(ssl_files):
+            sys.exit(1)
+
+        root_certificate, certificate, private_key = ssl_files
+
+        with open(root_certificate, "rb") as file:
+            root_certificate_b = file.read()
```

Comment:
Instead of naming these `_b`, why don't we rename e.g., `root_certificate` to `root_certificate_path`?